arrayRepeat.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * Creates a new array containing the elements of an existing array repeated
+ * @since 4.0.0
```

Comment:
This would also need to be updated I think, not sure what the correct version should be though.